### ISSUE TITLE
`Table.__init__` is not creating slugs

### ISSUE THREAD

**Issue by turicas** [turicas/rows#181]
To reproduce:

``` python
from collections import OrderedDict
import rows

table = rows.Table(fields=OrderedDict([
    ('Query Occurrence"( % ),"First Seen', rows.fields.FloatField),
]))
```

Exception raised:

```
ValueError: Type names and field names can only contain alphanumeric characters and underscores: 'Query Occurrence"( % ),"First Seen'
```

**Comment by humrochagf** [comment]
the best solution for that problem is to use a slugify helper function

i see 3 approaches for that:
1. keep it stdlib and follow Armin Ronacher`s guide to it http://flask.pocoo.org/snippets/5/
2. the middle approach to cover asian characters also at the Armin guide, but it uses external unidecode lib
3. go for a full external solution that works on any cases: https://github.com/un33k/python-slugify

**Comment by turicas** [comment]
@humrochagf there is a simple slug function inside the library: `rows.plugins.utils.slug`. There is also another issue to check a better solution for this slug task: https://github.com/turicas/rows/issues/116
This issue is related directly to using the slug function inside `Table.__init__`.

**Comment by humrochagf** [comment]
sorry i was looking at the wrong file

i made the PR #197 in order to fix the bug

**Comment by turicas** [comment]
Fixed in df91f54.
